bootstrap-page.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<head>
+    <title>Bootstrap Page</title>
+    <style>
+        .container {
+            float: left;
+            width: 50%;
+        }
+        .container-2 {
+            float: right;
+            width:50%;
+        }
+        .carbon-ad {
+            float:left;
+            width: 100%;
+            border: 3px;
+        }
+        .container-3 {
+            float: none;
+            width: 50%;
+        }
+        .footer {
+            width: 25%;
+        }
+    </style>
+</head>
+<body>
+    <div>
+        <nav>
+            <ul style="list-style:none;">
+                <li><a href=#"logo">Logo</a></li>
+                <li><a href=#"home">Home</a></li>
+                <li><a href=#"Docs">Docs</a></li>
+                <li><a href=#"Examples">Examples</a></li>
+                <li><a href=#"Icons">Icons</a></li>
+                <li><a href=#"Themes">Themes</a></li>
+                <li><a href=#"Blog">Blog</a></li>
+                <li><a href=#"Github">Github icon</a></li>
+                <li><a href=#"Twitter">Twitter icon</a></li>
+                <li><a href=#"Open-Collective">Open Collective icon</a></li>
+                <li><button type="button">Download</button></li>                
+            </ul>
+        </nav>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <h1>Build fast, responsive sites with Bootstrap</h1>
+                <p>Quickly design and customize responsive mobile-first sits with Bootstrap, the world’s most popular front-end open source toolkit, featuring Sass variables and mixins, responsive grid system, extensive prebuilt components, and powerful JavaScript plugins.</p>
+                <button type="button">Download Examples</button>
+                <button type="button">Download source code</button>
```

Comment:
When you want two things "side by side", you'd add a class to both of these, and then use either inline-block or float.

Also, put these two buttons as child of their own `div`, so that they go onto their own line.